hack/test/e2e-image-factory.sh

```diff
@@ -24,7 +24,7 @@ case "${FACTORY_BOOT_METHOD:-iso}" in
     QEMU_FLAGS+=("--ipxe-boot-script=${FACTORY_SCHEME}://${PXE_FACTORY_HOSTNAME}/pxe/${FACTORY_SCHEMATIC}/${FACTORY_VERSION}/metal-amd64")
     ;;
   secureboot-iso)
-    QEMU_FLAGS+=("--iso-path=${FACTORY_SCHEME}://${FACTORY_HOSTNAME}/image/${FACTORY_SCHEMATIC}/${FACTORY_VERSION}/metal-amd64-secureboot.iso" "--with-tpm2" "--encrypt-ephemeral" "--encrypt-state" "--disk-encryption-key-types=tpm")
```

Comment:
I'm not sure why it got removed here

Reply:
If we run all tests with TPM2 we don't need it here. However we shouldn't do it for all

Reply:
ah, I see it now

Reply:
we have at least in the `misc` section the test with UEFI disabled (BIOS mode), this could be a good place with disabled tpm2